index.js

```diff
@@ -90,3 +137,26 @@ const sendgridPropsMap = {
     postalCode: 'postal_code',
     postal_code: 'postal_code'
 }
+
+// parseCustomFieldsMap parses custom properties in a format like "myProp1=my_prop1,my_prop2".
+function parseCustomFieldsMap(customProps) {
+    const result = {}
+    if (customProps) {
+        customProps.split(',').forEach((prop) => {
+            const parts = prop.split('=')
+            if (parts.length == 1) {
+                result[parts[0]] = parts[0]
+            } else if (parts.length == 2) {
+                result[parts[0]] = parts[1]
+            } else {
+                throw new Error(`bad format in '${prop}'`)
```

Comment:
Nit: given we have all other errors starting with upper case, we could do this here too

Reply:
Done in e5978c1.